### ISSUE TITLE
[FireMonkey] New script inherits stored values from previously selected script

### ISSUE THREAD

**Issue by pintassilgo** [erosman/support#491]
Create this script:

```js
// ==UserScript==
// @name             FM Bug
// @match            https://www.example.com/?i491
// @version          1.0
// ==/UserScript==

GM.setValue('foo', 'bar');
```

Close editor, then open `https://www.example.com/?i491` to store the value.

Click FM toolbar button, click "FM Bug" then "Edit".

Click `+ JS` to create a new script, then paste the following code:

```js
// ==UserScript==
// @name             Other script
// @match            https://www.example.com/?otherpage
// @version          1.0
// ==/UserScript==

document.body.innerHTML += GM_getValue('foo');
```

Now open `https://www.example.com/?otherpage` and you'll see that the new script has `foo=bar` value.

Storage should be cleared in options page when you click to create a new script.

**Comment by erosman** [comment]
I see .. it is a bug that when `+ JS` is hit, User Metadata & Storage under 3-dot menu is not cleared.
Fixed for v2.57

**Comment by erosman** [comment]
v2.57 uploaded
